scripts/test_recovery_memeory.py

```diff
@@ -0,0 +1,42 @@
+from argparse import ArgumentParser
+
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+import matplotlib.pyplot as plt
+
+from cf_il.recover_memory import recovery_memory
+from datasets import SequentialCIFAR10
+from utils.args import add_management_args
+
+# Add args parser
+parser = ArgumentParser(description='cflr', allow_abbrev=False)
+
+add_management_args(parser)
+args = parser.parse_known_args()[0]
+
+# fake args
+setattr(args, "validation", True)
+setattr(args, "batch_size", 32)
+
+# Load dataset
+cif = SequentialCIFAR10(args)
+
+num_classes = cif.N_CLASSES_PER_TASK * cif.N_TASKS
+
+# Load model
+model = torch.hub.load('pytorch/vision:v0.10.0', 'resnet18', pretrained=True)  # False
```

Comment:
w artykule nie używali pretrenowanych sieci

Reply:
True, zostawiłem to testowo, ale trzeba zmienić